web/package.json

```diff
@@ -56,11 +57,14 @@
     "focus-trap-react": "^9.0.2",
     "htmlparser2": "^7.2.0",
     "i18next": "^20.6.1",
+    "license-checker": "^25.0.1",
     "lodash": "^4.17.21",
     "mapbox-gl": "npm:empty-npm-package@1.0.0",
     "maplibre-gl": "^2.1.9",
     "moment": "2.29.4",
     "normalize.css": "^8.0.1",
+    "npm-license-crawler": "^0.2.1",
+    "path-to-regexp": "^1.7.0",
```

Comment:
Why do we need this?

Reply:
I dont't know where this dependencies came from. I removed them and it still works fine